test/test.ml

```diff
@@ -264,6 +264,8 @@ let run library_name suites =
             ((suite.suite_name, outcomes)::aggregated_outcomes) rest
       end
   in
+  
+  Lwt_engine.set (new Lwt_engine.libuv ());
```

Comment:
It's ok to leave this for now, but we will probably need to remove it before merging (note to self). We might do something like factor out the Luv engine into a separate package, and give it a tester that does this call, then calls back into the ordinary lwt.unix tester.